web/template/create_document.gohtml

```diff
@@ -121,7 +120,11 @@
                 <div class="govuk-button-group govuk-!-padding-top-5">
                     <button class="govuk-button" data-module="govuk-button" type="submit" name="recipientControls" value="addNewRecipient">Continue</button>
                 </div>
-                <a data-app-iframe-cancel class="govuk-link govuk-link--no-visited-state" href="#">Cancel</a>
+                {{ $url := (printf "create-document?id=%d&case=%s&templateId=%s&hasViewedInserts=true" .Case.ID .Case.CaseType .TemplateSelected.TemplateId) }}
+                {{ range .SelectedInserts }}
+                    {{ $url = (printf "%s&insert=%s" $url .)  }}
+                {{ end }}
+                <a href="{{$url}}" class="govuk-link govuk-link--no-visited-state">Cancel</a>
```

Comment:
You might want to consider adding a `Back` property to `createDocumentData` and do this logic in Go rather than the templating language. Either way you also need to apply the prefix to it so it'll always link to the right place.